.github/workflows/test_development_versions.yml

```diff
@@ -1,4 +1,4 @@
-name: Development version tests
+name: Tests
```

Comment:
Changing this overall name (as well as for the other workflows) makes the summary of checks a bit odd because there are now 3 identically named sections. Can we instead, keep these names as they were? This makes the job names a hit redundant information but that way all ways of viewing it still make sense imo

Reply:
OK.  The redundancy is of course why all the jobs had the same secondary name.  It's too bad the "Settings" interface doesn't show the full name.